From 9319d9906312579486d23bd20a6cf7de4fc5f027 Mon Sep 17 00:00:00 2001
From: David Benjamin <davidben@google.com>
Date: Thu, 23 Mar 2023 00:22:54 -0400
Subject: [PATCH] Fix parameter name for i2d_X509_EXTENSION

alg was a copy-paste error.

Change-Id: I4c591bb98f7da853c79679727d49a00c7c65ad91
Reviewed-on: https://boringssl-review.googlesource.com/c/boringssl/+/58305
Reviewed-by: Alex Gaynor <alex.gaynor@gmail.com>
Reviewed-by: Bob Beck <bbe@google.com>
Commit-Queue: Bob Beck <bbe@google.com>
Auto-Submit: David Benjamin <davidben@google.com>
(cherry picked from commit 2e13e36e7477cfe2ef48312634b1c34103da4899)
---
 include/openssl/x509.h | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/include/openssl/x509.h b/include/openssl/x509.h
index d7f157190..c509ee1a6 100644
--- a/include/openssl/x509.h
+++ b/include/openssl/x509.h
@@ -1085,10 +1085,9 @@ OPENSSL_EXPORT X509_EXTENSION *d2i_X509_EXTENSION(X509_EXTENSION **out,
                                                   const uint8_t **inp,
                                                   long len);
 
-// i2d_X509_EXTENSION marshals |alg| as a DER-encoded X.509 Extension (RFC
+// i2d_X509_EXTENSION marshals |ex| as a DER-encoded X.509 Extension (RFC
 // 5280), as described in |i2d_SAMPLE|.
-OPENSSL_EXPORT int i2d_X509_EXTENSION(const X509_EXTENSION *alg,
-                                      uint8_t **outp);
+OPENSSL_EXPORT int i2d_X509_EXTENSION(const X509_EXTENSION *ex, uint8_t **outp);
 
 // X509_EXTENSION_dup returns a newly-allocated copy of |ex|, or NULL on error.
 // This function works by serializing the structure, so if |ex| is incomplete,
